vimdoc/module.py

```diff
@@ -112,44 +112,47 @@ def Close(self):
 
     All default sections that have not been overridden will be created.
     """
-    if self.GetCollection(vimdoc.FUNCTION) and 'functions' not in self.sections:
-      functions = Block(vimdoc.SECTION)
-      functions.Local(id='functions', name='Functions')
-      self.Merge(functions)
-    if (self.GetCollection(vimdoc.EXCEPTION)
-        and 'exceptions' not in self.sections):
-      exceptions = Block(vimdoc.SECTION)
-      exceptions.Local(id='exceptions', name='Exceptions')
-      self.Merge(exceptions)
-    if self.GetCollection(vimdoc.COMMAND) and 'commands' not in self.sections:
-      commands = Block(vimdoc.SECTION)
-      commands.Local(id='commands', name='Commands')
-      self.Merge(commands)
-    if self.GetCollection(vimdoc.DICTIONARY) and 'dicts' not in self.sections:
-      dicts = Block(vimdoc.SECTION)
-      dicts.Local(id='dicts', name='Dictionaries')
-      self.Merge(dicts)
+    # ----------------------------------------------------------
+    # Add default sections.
+
+    # type : (id, name)
+    default_sections = {
+        vimdoc.FUNCTION: ('functions', 'Functions'),
+        vimdoc.EXCEPTION: ('exceptions', 'Exceptions'),
+        vimdoc.COMMAND: ('commands', 'Commands'),
+        vimdoc.DICTIONARY: ('dicts', 'Dictionaries'),
+        vimdoc.FLAG: ('config', 'Configuration'),
+        vimdoc.SETTING: ('config', 'Configuration'),
+      }
+
+    for k, (id, name) in default_sections.items():
+      if self.GetCollection(k) and (id not in self.sections):
+        # Create the section if it does not exist.
+        block = Block(vimdoc.SECTION)
+        block.Local(id=id, name=name)
+        self.Merge(block)
+
+    # If any maktaba flags were documented, add a default configuration section
+    # to explain how to use them.
     if self.GetCollection(vimdoc.FLAG):
```

Comment:
Comment to explain that the order is determined by the listing above (if that's actually true). If not, we'll want to change it to preserve the ordering that the preexisting version had.